app/provider.tsx

```diff
@@ -72,6 +72,28 @@ export function Providers({ children }: { children: React.ReactNode }) {
         default: "#29282b",
       },
     },
+    components: {
+      MuiTabs: {
+        styleOverrides: {
+          // Surcharge les styles pour l'indicateur de l'onglet sélectionné
+          indicator: {
+            backgroundColor: "transparent",
+          },
+        },
+      },
+      MuiTab: {
+        styleOverrides: {
+          // Surcharge les styles pour les onglets non sélectionnés
```

Comment:
can you comment in english here ? we try to make it easy for all contributors to go through code